package.json

```diff
@@ -22,26 +22,25 @@
   "dependencies": {
     "@fortawesome/fontawesome-free": "^6.7.2",
     "@srcery-colors/srcery-palette": "^1.0.4",
```

Comment:
Guess we can remove this too?

Reply:
Please leave the palette, I was meaning to bring it up on discord, but I've not had time change yet: I think we should consider adding some css/sass vars to the palette file, so that we can once again rely on upstream for colors.